pkg/aws/v1alpha3/common_validator.go

```diff
@@ -81,6 +82,30 @@ func ValidateOrgNamespace(meta metav1.Object) error {
 
 	return nil
 }
+
+func ValidateOperatorVersion(meta metav1.Object) error {
```

Comment:
Here's the func which verifies operator version, either aws-operator version or cluster-operator version